.github/workflows/terraform.yml

```diff
@@ -0,0 +1,36 @@
+name: Terraform docs and formatting
+on:
+  pull_request_target:
+    types:
+      - opened
+      - synchronize
+      - reopened
+  push:
+  #  branches:
+  #    - master
+jobs:
+  formatting:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+      - name: terraform fmt
+        uses: dflook/terraform-fmt-check@fc6a4d63e251c5d6f247fc8310171a4e45e18210
+  docs:
+    needs: formatting
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+      - name: Render terraform docs inside the README.md and push changes back to PR branch
+        uses: terraform-docs/gh-actions@f6d59f89a280fa0a3febf55ef68f146784b20ba0
+        with:
```

Comment:
https://github.com/isovalent/terraform-aws-vpc/actions/runs/4301463160/jobs/7498723808#step:4:35

Can we switch that to https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/?

Reply:
I'm going to add on to this, could we please move this to a more recent commit that addresses this deprecated command? 

Please try out the latest commit: 
 `terraform-docs/gh-actions@cfde42f79b15256c71f4b79ae1d6acea0f689952`